module/common/Logging.cpp

```diff
@@ -0,0 +1,38 @@
+/**
+ * @file Logging.cpp
```

Comment:
ファイル名も`Logging.cpp`のままだから，`Logger.cpp`に直しておいて
```suggestion
 * @file Logger.cpp
```